src/pages/registration/DuplicateWarning.tsx

```diff
@@ -1,18 +1,17 @@
 import OpenInNewOutlinedIcon from '@mui/icons-material/OpenInNewOutlined';
-import { Box, Typography } from '@mui/material';
+import { Box, BoxProps, Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
 import { Link } from 'react-router-dom';
 import { StyledInfoBanner } from '../../components/styled/Wrappers';
 import { dataTestId } from '../../utils/dataTestIds';
-import { getRegistrationLandingPagePath } from '../../utils/urlPaths';
 
-interface DuplicateWarningProps {
+interface DuplicateWarningProps extends BoxProps {
```

Comment:
Siden det i praksis bare er `sx` som brukes, så burde kanskje bare sx være her også?
Ev. kan vi spreade alle BoxProps med noe sånt da `<Box ...props sx={{ (...), ...props.sx }}>`
```suggestion
interface DuplicateWarningProps extends Pick<BoxProps, "sx"> {
```